### PR TITLE
Sentence improvements to SECURITY.md

### PR DESCRIPTION
**Pull request by rzrymiak**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 # Security Policy
 
 Compiler Explorer allows remote users to compile and, if configured, execute code. We take security seriously, and
-encourage users to promptly report security vulnerabilities they find.
+encourage users to promptly report any possible security vulnerabilities they discover.
```
Comment (mattgodbolt) [inline SECURITY.md:4]: Thanks; this is a better phrasing.

File: SECURITY.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 
 ## Reporting a Vulnerability
 
-If the issue can be reported without revealing exploitable specifics, please file
+If the issue can be reported without revealing any exploitable specifics, please file
```
Comment (mattgodbolt) [inline SECURITY.md:8]: This seems ok. thanks

File: SECURITY.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 [an issue](https://github.com/compiler-explorer/compiler-explorer/issues/new/choose) as a bug.
 
 Please email matt@godbolt.org with specifics, or if the bug can't be reported publically without leaving an obvious
 exploit in the public eye.
 
 We expect to get back within a day or two. If you don't hear from us, please do ping us again, or reach out to us on the
-[Discord](https://discord.gg/wFXUwDp).
+[Discord](https://discord.gg/wFXUwDp) so that we can fix the bug promptly.
```
Comment (mattgodbolt) [inline SECURITY.md:15]: I'd rather not make any promises of promptness here.
```suggestion
[Discord](https://discord.gg/wFXUwDp).
```

**Review by mattgodbolt — COMMENTED** [review]
Mostly ok; but I don't want to sound like we always can look into things that promptly. If you have another suggested phrasing for the last point, please let us know!

**Comment by dkm** [comment]
I fear we won't hear from @rzrymiak . Should we discard this PR @mattgodbolt ?

**Comment by mattgodbolt** [comment]
I think so; thanks for reminding me @dkm . I'm going to close this. If we have more material changes to the policy we will revisit.

**Pull request closed without merging**